systems/edge-nodes/modules/caddy.nix

```diff
@@ -161,6 +162,9 @@
           }
           redir /phpBB/ https://forums.tgstation13.org/
           redir /phpBB/*.php* https://forums.tgstation13.org/{http.request.orig_uri.path.file}?{http.request.orig_uri.query}{http.request.orig_uri.path.*/}
+          handle_path /wiki/* {
+            redir * https://wiki.tgstation13.org{uri} permanent
```

Comment:
Is the missing slash intentional? Wiki has a trailing slash, is that how this is supposed to be?

Reply:
between the domain and URI? yes, the official caddy docs do this too